### PR TITLE
Add valid_docs rule handle multibyte characters.

### LINKED ISSUE #295: valid_docs rule doesn't handle multibyte characters
test case:
```sh
/// docs👨‍👩‍👧‍👧
/// - returns: false
public func no() -> Bool { return false }
```

### PR DESCRIPTION
**Pull request by norio-nomura**
Fix #295

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,10 @@
   [JP Simard](https://github.com/jpsim)
   [#279](https://github.com/realm/SwiftLint/issues/279)
 
+* valid_docs rule doesn't handle multibyte characters.
```
Comment (jpsim) [inline CHANGELOG.md]: This should describe the fix rather than the previous bug: ```The `valid_docs` rule now handles multibyte characters.```.

**Comment by jpsim** [comment]
Other than the changelog entry, :+1:. Nice catch.

**Comment by norio-nomura** [comment]
:pray:

**Comment by jpsim** [comment]
This needs to be rebased

**Comment by norio-nomura** [comment]
I rebased. :bow:

**Pull request merged**